app/controllers/api/v1/hotels_controller.rb

```diff
@@ -0,0 +1,68 @@
+class Api::V1::HotelsController < ApplicationController
+  
+  def create
+    activity_type = ActivityType.find_by(trip_id: params[:trip])
+    
+    if activity_type.activities != []
+      lats = []
+      longs = []
+      activity_type.activities.each do |i|
+        lat = i['lat']
+        lng = i['lng']
+        
+        lats << lat.to_f
+        longs << lng.to_f
```

Comment:
`i` is not a great variable name, `activity`?

Could be shortened to below but up to you
```
lats << activity['lat'].to_f
longs << activity['lng'].to_f
```

Reply:
changed to activities, and refactored the push into lats, longs arrays